src/db.c

```diff
@@ -376,7 +375,9 @@ kvobj *dbAddRDBLoad(redisDb *db, sds key, robj **valref, long long expire) {
  *   update of a value of an existing key (when false).
  * - The `link` is optional, can save lookup, if provided.
  */
-static void dbSetValue(redisDb *db, robj *key, robj **valref, int overwrite, dictEntryLink link) {
+static void dbSetValue(redisDb *db, robj *key, robj **valref, int notifyOverwrite, 
```

Comment:
i don't like `notifyOverwrite`, just as the comment mentioned: 
```
 * - The 'overwrite' flag is an indication whether this is done as part of a
 *   complete replacement of their key, which can be thought as a deletion and
 *   replacement (in which case we need to emit deletion signals), or just an
 *   update of a value of an existing key (when false).
```
If it is used in non-notify in the future, it will become strange.

Reply:
reverted. thanks :)